plugin/pkg/namespacedcloudprofile/validator/admission.go

```diff
@@ -320,7 +324,28 @@ func validateNamespacedCloudProfileExtendedMachineImages(machineVersion gardenco
 	return allErrs
 }
 
-func (c *validationContext) validateSimulatedCloudProfileStatusMergeResult(_ admission.Attributes) error {
+func (c *validationContext) validateLimits(limitsPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if c.namespacedCloudProfile.Spec.Limits == nil ||
+		c.parentCloudProfile.Spec.Limits == nil {
+		return nil
+	}
```

Comment:
```suggestion
	if c.namespacedCloudProfile.Spec.Limits == nil || c.parentCloudProfile.Spec.Limits == nil {
		return nil
	}
```

Nit - improves readability if it's in the same line